user-files/simulations/gCloud11Application/UpdateSupplerDetails.scala

```diff
@@ -16,87 +16,71 @@ object UpdateSupplerDetails {
 
 	val headers_7 = Map("Accept" -> "*/*")
 
-	val updateSupplierDetails = exec(http("update_details_request_1")
+    var sp = System.getProperty("scalePauses").toDouble
```

Comment:
Weird whitespace here (and on the declaration equivalent)? Not a blocker

Reply:
Ah might be an artefact of a tricky rebase I did.

Reply:
Ahh no it's because I didn't notice these scala files are tab-indented, this is space indented.